index.html

```diff
@@ -1,241 +1,329 @@
-                        <p class="language">Javascript</p>
-                    </li>
-                    <!-- HTML icon and name area -->
-                    <li class="language-container lang-box">
-                        <img src="./images/Html.png" class="lang-img" alt="Html">
-                        <p class="language css-html-lng">HTML</p>
-                    </li>
-                    <!-- CSS icon and name area -->
-                    <li class="language-container lang-box">
-                        <img src="./images/css.png" class="lang-img" alt="css">
-                        <p class="language css-html-lng">CSS</p>
-                    </li>
-                    <!-- FRAMEWORKS AND SKILLS -->
-                    <li class="frame-Skill">
-                        <div class="frame-skl-container">
-                            <p class="frOrSkill">Frameworks</p>
-                            <i class="fa fa-angle-right" aria-hidden="true"></i>
-                        </div>
-                        <hr>
-                        <!-- SKILLS -->
-                        <div class="frame-skl-container">
-                            <p class="frOrSkill">Skills</p>
-                            <i class="fa fa-angle-right" aria-hidden="true"></i>
-                        </div>
-                        <hr>
-                    </li>
+          </div>
+        </div>
+      </section>
+      <!-- ABOUT SELF SECTION -->
+      <section class="about-section-container">
+        <div class="about-section-card">
+          <div class="lang-left">
+            <h2 class="name-intro">
+              About <br />
+              Myself
+            </h2>
+            <p class="personal-info">
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <div class="connect"><a href="#">LET'S CONNECT</a></div>
+            <ul class="social">
+              <li>
+                <a href="#"
+                  ><img src="./images/twitter.svg" alt="Twitter Icon"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/linkedin.svg" alt="LinkedIn icon"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/medium.svg" alt="Medium icon"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/github.svg" alt="Github icon"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/angellist.svg" alt="Angellist icon"
+                /></a>
+              </li>
```

Comment:
- [ ] [Optional] Kindly add `align-items: center;` to your class `social` so as to create a better alignment for your icons